src/plugins/intel_cpu/src/nodes/executors/fullyconnected_implementations.cpp

```diff
@@ -335,6 +355,62 @@ const std::vector<ExecutorImplementation<FCAttrs>>& getImplementations() {
                 return std::make_shared<ShlFCExecutor>(attrs, postOps, memory, context);
             }
         )
+        OV_CPU_INSTANCE_X64(
+            "matmul",
+            ExecutorType::Dnnl,
+            OperationType::MatMul,
+            ShapeTolerance::Dependant,
+            // supports
+            [](const FCConfig& config) -> bool {
+                // CPU_DEBUG_CAP_ENABLE(if (!std::getenv("OV_CPU_USE_MAMTUL_OVER_FC")) { return false; });
```

Comment:
uncomment before merge.
Implement some static getenv function to avoid getting variable all the time.